tests/TestableIO.System.IO.Abstractions.Wrappers.Tests/ApiParityTests.cs

```diff
@@ -7,7 +7,7 @@
 
 namespace System.IO.Abstractions.Tests
 {
-    [TestFixture]
+    [TestFixture(IgnoreReason = "TODO: temporarily disable in #906 as it is unclear how the parity tests are updated.")]
```

Comment:
This needs to be fixed before merging this PR!

Reply:
Each mismatched snapshot will be created in a `__mismatch__` directory below https://github.com/TestableIO/System.IO.Abstractions/tree/main/tests/TestableIO.System.IO.Abstractions.Wrappers.Tests/__snapshots__.

Just overwrite files from the `__snapshots__` directory with the mismatched ones to acknowledge them.

See https://swisslife-oss.github.io/snapshooter/docs/snapshot-mismatch-handling

Reply:
Thanks for the hint, I re-enabled the parity tests. Please note, that I had to change some mappings in [ApiParityTests](https://github.com/TestableIO/System.IO.Abstractions/pull/906/commits/4da6ca728b4b455e22a63864374383874c1b54c1#diff-3c3ddade875d30a4c4d1b8088475ca0128dc09de21ae6dda4b3e913d1ab81a92) due to the changed return values:
- `FileSystemStream` instead of `Stream`
- `IWaitForChangedResult` instead of `WaitForChangedResult`